### PR TITLE
Improve tracing logs a bit and refactor sx127x register writes

### PR DESCRIPTION
**Pull request by plaes**

### CODE REVIEW

File: lora-phy/src/sx1276_7_8_9/mod.rs [hunk 1]
```diff
@@ -55,14 +55,9 @@ where
     }
 
     // Utility functions
-    async fn write_register(
-        &mut self,
-        register: Register,
-        value: u8,
-        is_sleep_command: bool,
-    ) -> Result<(), RadioError> {
+    async fn write_register(&mut self, register: Register, value: u8) -> Result<(), RadioError> {
```
Comment (lucasgranberg) [inline lora-phy/src/sx1276_7_8_9/mod.rs:58]: are you using rustfmt? I don't really care how it looks as long as we don't go back and forth every commit

**Review by lucasgranberg — APPROVED** [review]
Much cleaner this way, lgtm!

**Review by lthiery — APPROVED** [review]

File: lora-phy/src/sx1276_7_8_9/mod.rs (hunk 1 above)
Comment (plaes) [reply lora-phy/src/sx1276_7_8_9/mod.rs:58]: Well, after dropping `is_sleep_command: bool,` this fits nicely on single line...

Comment (lucasgranberg) [reply lora-phy/src/sx1276_7_8_9/mod.rs:58]: yeah I noticed that 1 second after commenting, sry

**Pull request merged**